daffodil-core/src/main/scala/org/apache/daffodil/dpath/DFDLExpressionParser.scala

```diff
@@ -52,14 +53,14 @@ class DFDLPathExpressionParser[T <: AnyRef](
   namespaces: NamespaceBinding,
   context: DPathCompileInfo,
   isEvaluatedAbove: Boolean,
-  host: OOLAGHost) extends RegexParsers {
+  host: OOLAGHost with HasTunable) extends RegexParsers {
```

Comment:
Maybe this OOLAGHost with HasTunable should just become SchemaComponent.

Reply:
Would be nice if we didn't need the ``with HasTunable`` all over the place.

Reply:
Is now BasicComponent defined in runtime1 module.